js/forum/dist/extension.js

```diff
@@ -62,6 +63,27 @@ System.register("wuethrich44/sso/main", ["flarum/extend", "flarum/app", "flarum/
                     items.remove('changeEmail');
```

Comment:
No, I'm not simply removing the entire section. The new function is triggered after your `removeProfileActions` function. I check if it is empty. If so, remove, if not, do nothing.